### ISSUE TITLE
Regression in latest version around array length validation

### ISSUE THREAD

**Issue by Miradorn** [martinthenth/goal#114]
hey there again!

Sorry but I think I've found another regression while trying to update goal to the latest version:

```
defmodule Params do
  use Goal

  defparams params
    required :objects, {:array, :map}, min: 1, max: 10
  end
end
```

than use it like

```
Params.validate(:params, %{objects: []}) => {:ok, ...}
```

I think this should (and did before.) raise, since the objects list is less than 1.
Same happens when passing in a list of >10.

Thanks for any help!

**Comment by martinthenth** [comment]
Hi @Miradorn! Thanks for the report. The issue should be fixed in version `1.1.2` and I added tests to confirm. Could you verify it works on your end? If the issue persists, feel free to reopen this ticket. 🙂

**Comment by Miradorn** [comment]
works beautifully! Thanks a lot ❤️
